phpstan-baseline.neon

```diff
@@ -0,0 +1,1186 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Argument of an invalid type array\\|null supplied for foreach, only iterables are supported\\.$#"
+			count: 1
+			path: src/Adapter/BaseAdapter.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Adapter\\\\BaseAdapter\\:\\:getOptions\\(\\) return type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/BaseAdapter.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Adapter\\\\BaseAdapter\\:\\:getTimezoneOffset\\(\\) has no return type specified\\.$#"
+			count: 1
+			path: src/Adapter/BaseAdapter.php
+
+		-
+			message: "#^Property ipl\\\\Sql\\\\Adapter\\\\BaseAdapter\\:\\:\\$options type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/BaseAdapter.php
+
+		-
+			message: "#^Property ipl\\\\Sql\\\\Adapter\\\\BaseAdapter\\:\\:\\$quoteCharacter type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/BaseAdapter.php
+
+		-
+			message: "#^Property ipl\\\\Sql\\\\Adapter\\\\Mssql\\:\\:\\$quoteCharacter type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/Mssql.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Adapter\\\\Mysql\\:\\:getOptions\\(\\) return type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/Mysql.php
+
+		-
+			message: "#^Property ipl\\\\Sql\\\\Adapter\\\\Mysql\\:\\:\\$quoteCharacter type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Adapter/Mysql.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\CommonTableExpressionInterface\\:\\:getWith\\(\\) return type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/CommonTableExpressionInterface.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Compat\\\\FilterProcessor\\:\\:assembleFilter\\(\\) has no return type specified\\.$#"
+			count: 1
+			path: src/Compat/FilterProcessor.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Compat\\\\FilterProcessor\\:\\:assembleFilter\\(\\) has parameter \\$level with no type specified\\.$#"
+			count: 1
+			path: src/Compat/FilterProcessor.php
+
+		-
+			message: "#^Method ipl\\\\Sql\\\\Compat\\\\FilterProcessor\\:\\:assemblePredicate\\(\\) has no return type specified\\.$#"
+			count: 1
+			path: src/Compat/FilterProcessor.php
+
+		-
+			message: "#^Parameter \\#1 \\$haystack of function strpos expects string, mixed given\\.$#"
+			count: 1
+			path: src/Compat/FilterProcessor.php
+
+		-
+			message: "#^Parameter \\#3 \\$subject of function str_replace expects array\\|string, mixed given\\.$#"
+			count: 2
+			path: src/Compat/FilterProcessor.php
+
+		-
+			message: "#^Part \\$column \\(array\\<string\\>\\|string\\) of encapsed string cannot be cast to string\\.$#"
```

Comment:
*encapsed*: is this a typo yours or by phpstan devs? :joy:

Reply:
phpstan

Reply:
Actually it's not a typo

Reply:
Well, not an unintended typo, yes. But a [non-existent word](https://www.dict.cc/?s=encapsed) anyway.